.changeset/empty-flies-join.md

```diff
@@ -0,0 +1,177 @@
+---
+"@gradio/client": major
+"gradio_client": major
+---
+
+highlight:
+
+#### Clients 1.0 Launch! 
+
+We're excited to unveil the first major release of the Gradio clients.
+We've made it even easier to turn any Gradio application into a production endpoint thanks to the clients' **ergonomic**, **transparent**, and **portable** design.
+
+#### Ergonomic API 💆
+
+**Stream From a Gradio APP in 5 lines**
+
+Use the `submit` method to get a job you can iterate over:
```

Comment:
I think we should have a "regular" `.predict()` example first before an iterator example?